src/app/markdown/examples/markdown-example.component.scss

```diff
@@ -0,0 +1,4 @@
+
+.container {
+  width: 1000px;
```

Comment:
The editor takes 100% of it's container, this is just the editor container in example page.